packages/expo-updates/e2e/__tests__/Updates-e2e.test.ts

```diff
@@ -12,14 +12,23 @@ import { copyBundleToStaticFolder } from './utils/update';
 const SERVER_HOST = process.env.UPDATES_HOST;
 const SERVER_PORT = parseInt(process.env.UPDATES_PORT, 10);
 
-// Keep in sync with the manifest in .github/workflows/updates-e2e.yml
 const RUNTIME_VERSION = '1.0.0';
 
 const TIMEOUT_BIAS = process.env.CI ? 10 : 1;
 
+const repoRoot = process.env.EXPO_REPO_ROOT ?? process.env.GITHUB_WORKSPACE;
```

Comment:
I noticed this bit of logic (`process.env.EXPO_REPO_ROOT ?? process.env.GITHUB_WORKSPACE`) in a couple of places. While I don't think it's a big issue, would it work to make the GHA workflow define `EXPO_REPO_ROOT=$GITHUB_WORKSPACE` (using `$GITHUB_ENV`) and having the JS files read only EXPO_REPO_ROOT?